### PR TITLE
fix 10s delay for edge request when the node is offline

### PR DESCRIPTION
**Pull request by wackxu**
**What type of PR is this?**

Add one of the following kinds:
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes # part of #4423 

fix 10s delay for edge request when the node is offline

when the node if offline, the request for metaserver will generate the application and apply the application to the cloudcore, 
in the apply func, we use the sendSync function and will wait max time 10s to get the result

```
resp, err := beehiveContext.SendSync(edgemodule.EdgeHubModuleName, *msg, 10*time.Second)
```
but due to the network is lost between edgecore and cloudcore and the apply will spend 10s every time when process the request and this will affect code processing, such as edgemesh, will have a 10s deplay for get and list request when the node is offline. So if the node is offline, we do not generate the application since it is useless

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
LGTM

File: edge/pkg/metamanager/process.go [hunk 1]
```diff
@@ -485,8 +470,6 @@ func (m *metaManager) process(message model.Message) {
 		m.processQuery(message)
 	case model.ResponseOperation:
 		m.processResponse(message)
-	case messagepkg.OperationNodeConnection:
```
Comment (vincentgoat) [inline edge/pkg/metamanager/process.go:488]: Also can remove the code on the sender side?

Comment (wackxu) [reply edge/pkg/metamanager/process.go:488]: done

**Review by vincentgoat — COMMENTED** [review]
/lgtm

**Pull request merged**